pallets/ah-migrator/src/lib.rs

```diff
@@ -208,12 +223,15 @@ pub mod pallet {
 		+ pallet_vesting::Config
 		+ pallet_indices::Config
 		+ pallet_conviction_voting::Config
-		+ pallet_bounties::Config
-		+ pallet_treasury::Config
 		+ pallet_asset_rate::Config
 		+ pallet_timestamp::Config<Moment = u64> // Needed for testing
 		+ pallet_ah_ops::Config
+		+ pallet_claims::Config // Not on westend
+		+ pallet_bounties::Config // Not on westend
+		+ pallet_treasury::Config // Not on westend
```

Comment:
how do we handle this? removing after porting?

Reply:
This is removed by the `just port` command.  
It is a bit ugly, but not sure what better was Rust offers. Maybe some blanket trait impl or so, will try again after merging.